lib/tcp.js

```diff
@@ -157,7 +157,10 @@ exports.client = function (options, transportUtil) {
       reconnect.on('disconnect', function (err) {
         seneca.log.debug('client', type, 'disconnect', spec, topic, clientOptions,
            (err && err.stack) || err)
-        _.find(connections, { clientOptions: clientOptions }).setup = false
+        var connections = _.find(connections, { clientOptions: clientOptions });
```

Comment:
@friedemannsommer when using tcp with express, and after adding your fix, instead of the error, now we have a timeout. do you have the same problem?